examples/vmi-with-generic-hook.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
+metadata:
+  annotations:
+    hooks.kubevirt.io/hookSidecars: '[{"args": ["--version", "v1alpha2"], "image":
+      "registry:5000/kubevirt/sidecar-shim:devel", "configMap": {"name": "my-config-map",
+      "key": "my_script.sh"}}]'
```

Comment:
It would be nice to have the config map included too, not sure if doable. (Not a blocker imho)